sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Do you want to lock this at 2.70.0, so it will still use this version as newer versions are released? Or you could alternatively say at least this version, but newer ones if available? ( <code>sagemaker&gt;=2.70.0</code> ) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
`!pip install --force-reinstall sagemaker>=2.70.0` does not work with the `>`.  2.70 is fine and updates to SageMaker should not affect Training Compiler given that its components are in a DLC independent of SM Python SDK